WindowsDevicePortalWrapper/WindowsDevicePortalWrapper.Shared/IoT/WindowsUpdate.cs

```diff
@@ -44,6 +55,35 @@ public partial class DevicePortal
             return await this.Get<UpdateInstallTimeInfo>(InstallTimeApi);
         }
 
+        /// <summary>
+        /// Sets SoftAp Settings.
+        /// </summary>
+        /// <returns>Task tracking completion of the REST call.</returns>
+        public async Task SetUpdateInstallTime()
```

Comment:
Does this need an argument? I'm not sure what it's setting (does it trigger it immediately?)